command/push.go

```diff
@@ -72,36 +75,47 @@ func doPush(ctx *cli.Context) {
 		priority = conf.DefaultPriority
 	}
 
-	msg := models.MessageExternal{
-		Message:  msgText,
-		Title:    title,
-		Priority: priority,
+	parsedURL, err := url.Parse(stringURL)
+	if err != nil {
+		utils.Exit1With("invalid url", stringURL)
+		return
 	}
 
-	msg.Extras = map[string]interface{}{
-	}
+	var sent bool
+	for msgText := range msgText {
+		if !ctx.Bool("disable-unescape-backslash") {
+			msgText = utils.Evaluate(msgText)
+		}
 
-	if contentType != "" {
-		msg.Extras["client::display"] = map[string]interface{}{
-			"contentType": contentType,
+		msg := models.MessageExternal{
+			Message:  msgText,
+			Title:    title,
+			Priority: priority,
 		}
-	}
 
-	if clickUrl != "" {
-		msg.Extras["client::notification"] = map[string]interface{}{
-			"click": map[string]string{
-				"url": clickUrl,
-			},
+		msg.Extras = map[string]interface{}{}
+
+		if contentType != "" {
+			msg.Extras["client::display"] = map[string]interface{}{
+				"contentType": contentType,
+			}
 		}
-	}
 
-	parsedURL, err := url.Parse(stringURL)
-	if err != nil {
-		utils.Exit1With("invalid url", stringURL)
-		return
-	}
+		if clickUrl != "" {
+			msg.Extras["client::notification"] = map[string]interface{}{
+				"click": map[string]string{
+					"url": clickUrl,
+				},
+			}
+		}
```

Comment:
The "extras" creation could be moved outside of the for loop, so that it only has to be initialized once.